settings.lua

```diff
@@ -27,8 +27,22 @@ data:extend({
     default_value = true,
     order = "2",
   },
+  {
+    type = "int-setting",
+    name = "cube-cubecam-scale",
+    setting_type = "runtime-per-user",
+    default_value = 100,
+    order = "3",
+  }
 })
 
+local allowed_values_cubecam_scale = {}
```

Comment:
a dropdown with 200 values seems a bit unwieldy. I don't think we can make it a slider (unfortunately), maybe we should go in increments of 10% from 50% to 150% (I suspect outside roughly that range to be too small/big to be useful).

Reply:
it really depends on the player resolution. changed it to 10 - 150 %. I use a scale of ~60% and those small increments do matter.